src/app/@ansyn/ol/plugins/annotations/annotations.visualizer.ts

```diff
@@ -585,7 +585,7 @@ export class AnnotationsVisualizer extends EntitiesVisualizer {
 
 	labelTranslateMode(featureId: any) {
 		let oldFeature = null;
-		let event = null;
```

Comment:
Is this change needed?

Reply:
i guess, it come from master merge

Reply:
What do you mean? There is here a change from `let event = null` to `let event = undefined`. What is the reason for this change?

Reply:
i remember. 
there is another place that use `event=undefined`,
and there is check of `event === undefined` in some place to decide if
the measure is active or not.
it was or change the above or the other to null.